### PR TITLE
Deletion of cells and renaming of tensorflow to pytorch

### PR DESCRIPTION
**Pull request by tagekezo**
Deleted cells under label encoding and tokenization to avoid confusion
The following section under Dummy encode the labels is a little confusing,
df['CATEGORY'][1] retrieves the dataframe with index = 1; as the dataframe has been randomized, index=1 does not refer to the second row and hence does not match with encoded_y

Renamed tensorflow to pytorch for pytorch alternative notebooks

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: pytorch_alternatives/custom_pytorch_nlp/Headline Classifier Local.ipynb [hunk 1]
```diff
@@ -149,24 +149,6 @@
...
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "encoded_y[0]"
-   ]
-  },
```
Comment (athewsey) [inline pytorch_alternatives/custom_pytorch_nlp/Headline Classifier Local.ipynb:169]: Would prefer to fix this by correcting the code to pull out the relevant entry of `df`, rather than removing it... I think `.iloc[0]` would do it?

File: pytorch_alternatives/custom_pytorch_nlp/Headline Classifier Local.ipynb [hunk 2]
```diff
@@ -188,24 +170,6 @@
...
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "processed_docs[0]"
-   ]
-  },
```
Comment (athewsey) [inline pytorch_alternatives/custom_pytorch_nlp/Headline Classifier Local.ipynb:208]: As above, would be nice to fix rather than delete

File: pytorch_alternatives/custom_pytorch_nlp/Headline Classifier Local.ipynb [hunk 3]
```diff
@@ -514,9 +478,9 @@
  "metadata": {
   "instance_type": "ml.t3.medium",
   "kernelspec": {
-   "display_name": "Python 3 (PyTorch 1.6 Python 3.6 CPU Optimized)",
+   "display_name": "Python 3",
    "language": "python",
-   "name": "python3__SAGEMAKER_INTERNAL__arn:aws:sagemaker:ap-southeast-1:492261229750:image/pytorch-1.6-cpu-py36-ubuntu16.04-v1"
+   "name": "python3"
```
Comment (athewsey) [inline pytorch_alternatives/custom_pytorch_nlp/Headline Classifier Local.ipynb:483]: I assume this is incidental to whatever environment you were working on, rather than actively intending to change the kernel specs for the notebooks - should be restored to previous

**Review by athewsey — COMMENTED** [review]
Welcome & thanks very much for bringing these issues to our attention!

There are a few changes I'd like to see: I've highlighted some in the review, and would additionally mention:

- The 'SageMaker' and 'Local Mode' notebooks should be changed in step for the sections that are shared between them, so they continue to mirror each other except for changes explicitly related to using SageMaker
- The shuffling issue actually affects the equivalent Keras NLP samples too! So again, we should edit those files as well.

I already loaded up the changes to review them, and it's your first time contributing anyway - so I'll go ahead accept this PR and add additional commits to address the comments myself. 😄

**Comment by athewsey** [comment]
(Obviously if my edits didn't fully capture what you were trying to do, would be happy to discuss further!)
